internal/pkg/configure/tftp.go

```diff
@@ -27,7 +27,7 @@ func TFTP() error {
 	}
 
 	fmt.Printf("Writing PXE files to: %s\n", tftpdir)
-	for _, f := range [4]string{"x86.efi", "i386.efi", "i386.kpxe", "arm64.efi"} {
+	for _, f := range [4]string{"x86_64.efi", "x86_64.kpxe", "arm64.efi"} {
 		err = util.SafeCopyFile(path.Join(buildconfig.DATADIR(), "warewulf", "ipxe", f), path.Join(tftpdir, f))
```

Comment:
Can we just not copy all files from `$(WWDATADIR)/ipxe/` to the right place, instead of having a explicit file list like we have now?

Reply:
Yep, totally agree. Do you want to do that when you move the ipxe-update.sh into the Makefile? ... BTW, I'm happy to do that too, just let me know.